infra/auth0/text.tf

```diff
@@ -0,0 +1,713 @@
+
+
+resource "auth0_prompt_custom_text" "nb_email_verification" {
```

Comment:
In the future I think we'll put this in a json file and for_each this to avoid all the code, but ok for now